src/db/group.rs

```diff
@@ -16,15 +16,15 @@ pub(crate) type NodeLocation = Vec<Uuid>;
 
 pub enum SearchField {
     #[cfg(any(test, feature = "_merge"))]
-    UUID,
+    Uuid,
```

Comment:
This is a breaking change. I see that the PR was already merged, but it should probably have been pointed out in the commit description.

Reply:
I created https://github.com/sseemayer/keepass-rs/pull/254 since I don't think this field should even be used outside of this create.